core/client/app/controllers/team/user.js

```diff
@@ -15,6 +15,8 @@ export default Ember.Controller.extend({
 
     isNotOwnersProfile: Ember.computed.not('user.isOwner'),
 
+    isAdminUserOwnerProfile: Ember.computed.and('currentUser.isAdmin', 'user.isOwner'),
```

Comment:
Can you change this to be `isAdminUserOnOwnerProfile` - with the `on` in the middle, so it's clearer what it means? It would also be good to add just a couple of comment throughout this to explain what's happening and why.